### ISSUE TITLE
Nice work, will you add YOLACT++?

### ISSUE THREAD

**Issue by Maxinho96** [leohsuofnthu/Tensorflow-YOLACT#4]
I am very interested in your project. I would like to contribute but I am new to the field, maybe in the future.
Are you planning to support YOLACT++ and other backbones in the future?
Thank you and good work.

**Comment by leohsuofnthu** [comment]
Hi, Thanks for noticing my work. Right now I am focusing on providing a pre-trained weight for this model. Since the training speed is a bit slow now (sorry for that), I am now working on optimizing the computational time for loss and backpropagation. Once I finish this, I 'll start to train the model.  I hope I can finish that before the end of this month, cuz school is a bit busy too.  It would be nice to have ur contribution, right now I have work on the inference part (fastNMS, and evaluation). so maybe we could discuss that what u want to contribute :) I hope I can provide yolact ++ in the future !

**Comment by Maxinho96** [comment]
I would be happy to contribute to the inference part, but as I said I am new to the field so I don't know if I could be able to do that. I am also busy with classes and thesis work, so if I will have time I will let you know!
